x-pack/qa/rolling-upgrade/build.gradle

```diff
@@ -119,6 +119,8 @@ for (Version version : bwcVersions.wireCompatible) {
         setting 'xpack.security.enabled', 'true'
         setting 'xpack.security.transport.ssl.enabled', 'true'
         setting 'xpack.security.authc.token.enabled', 'true'
+        setting 'xpack.security.authc.token.timeout', '60m'
+        setting 'logger.org.elasticsearch.xpack.security.authc.TokenService', 'trace'
```

Comment:
Do we want this set to trace consistently or was it added for debug ?

Reply:
I would propose we set this to `trace` permanently. Given the added complexity of the implementation, I believe it would greatly help a test failure debugging.
I am not a master in the test infra, but I believe that when the test completed successfully the logs of the qa fixture are gone.

Reply:
I will echo @jaymode 's comment on https://github.com/elastic/elasticsearch/pull/40262#discussion_r270018422, I think it's a valid point

Reply:
_logger.org.elasticsearch.xpack.security.authc.TokenService_ is narrower than _logger.org.elasticsearch.xpack.security.authc_ . I don't really know, what to choose...